### PR TITLE
i18n(fr): Update Getting Started and Glossary

### PR DESCRIPTION
**Pull request by agriffard**
#### What kind of changes does this PR include?

- Translated content

#### Description

- Translate https://docs.astro.build/fr/getting-started/ from en page.
- A few changes in the French Glossary Readme.

### CODE REVIEW

**Comment by MoustaphaDev** [comment]
Suggested edit:
```diff

```

**Comment by MoustaphaDev** [comment]
Suggested edit:
```diff

```

**Comment by agriffard** [comment]
@MoustaphaDev I don't see anything in your 'Suggested edits'.

**Review by thomasbnt — APPROVED** [review]
ça me parait bien correct comme modification!

**Comment by MoustaphaDev** [comment]
> @MoustaphaDev I don't see anything in your 'Suggested edits'.

Ah désolé j'essayais de review ce PR avec l'extension github de vscode, ça marche pas on dirait.
Je vais envoyer mes suggestions

File: src/pages/fr/getting-started.md [hunk 1]
```diff
@@ -2,107 +2,108 @@
...
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  # create a new project with pnpm
+  pnpm create astro@latest
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  # create a new project with yarn
```
Comment (MoustaphaDev) [inline src/pages/fr/getting-started.md:59]:
```suggestion
  # créer un nouveau project avec npm
  npm create astro@latest
  ```
  </Fragment>
  <Fragment slot="pnpm">
  ```shell
  # créer un nouveau project avec pnpm
  pnpm create astro@latest
  ```
  </Fragment>
  <Fragment slot="yarn">
  ```shell
  # créer un nouveau project avec yarn
  ```

File: src/pages/fr/getting-started.md [hunk 2]
```diff
@@ -2,107 +2,108 @@
...
-
-*... et encore plus dans la catégorie **Fonctionnalités***
+## Démarrez votre premier projet
+
+Obtenez un nouveau projet Astro opérationnel localement avec notre assistant utile en ligne de commande `create-astro` !
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  # create a new project with npm
```
Comment (MoustaphaDev) [inline src/pages/fr/getting-started.md:47]:
```suggestion
  # créer un nouveau projet avec npm
```

File: src/pages/fr/getting-started.md [hunk 3]
```diff
@@ -2,107 +2,108 @@
...
+  npm create astro@latest
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  # create a new project with pnpm
```
Comment (MoustaphaDev) [inline src/pages/fr/getting-started.md:53]:
```suggestion
  # créer un nouveau projet avec pnpm
```

File: src/pages/fr/getting-started.md (hunk 1 above)
Comment (MoustaphaDev) [inline src/pages/fr/getting-started.md:59]: # créer un nouveau projet avec yarn

Comment (MoustaphaDev) [inline src/pages/fr/getting-started.md:59]:
```suggestion
  # créer un nouveau projet avec yarn
```

File: src/pages/fr/getting-started.md [hunk 4]
```diff
@@ -2,107 +2,108 @@
...
+💬 Partagez ce sur quoi vous avez travaillé dans notre channel `#showcase` !
 
-[Blog d'Astro](https://astro.build/blog/)
+## En savoir plus
 
-[Note de Mise à Jour d'Astro](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md)
+[Blog Astro](https://astro.build/blog/)
 
-[Guide de Migration d'Astro](/fr/migrate/)
+[Changelog Astro](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md)
```
Comment (MoustaphaDev) [inline src/pages/fr/getting-started.md:104]:
```suggestion

[Guide de Migration Astro](/fr/migrate/)

```

Comment (agriffard) [reply src/pages/fr/getting-started.md:104]: Migrate page does not exist yet in French.

File: src/pages/fr/getting-started.md (hunk 2 above)
Comment (agriffard) [reply src/pages/fr/getting-started.md:47]: I didn't want to translate code comment.

File: src/pages/fr/getting-started.md (hunk 4 above)
Comment (MoustaphaDev) [reply src/pages/fr/getting-started.md:104]: I think we can reference them anyway, it automatically appends "(EN)" to the text of the anchor and redirects to the english version.
It was done [here](https://github.com/withastro/docs/blob/main/src/pages/fr/reference/configuration-reference.md?plain=1#L174 ) and [here](https://github.com/withastro/docs/blob/main/src/pages/fr/concepts/islands.md?plain=1#L64)

File: src/pages/fr/getting-started.md (hunk 2 above)
Comment (MoustaphaDev) [reply src/pages/fr/getting-started.md:47]: Don't you think it would be helpful though?

Comment (agriffard) [reply src/pages/fr/getting-started.md]: Ok, done.

File: src/pages/fr/getting-started.md (hunk 3 above)
Comment (agriffard) [reply src/pages/fr/getting-started.md]: Done

File: src/pages/fr/getting-started.md (hunk 1 above)
Comment (agriffard) [reply src/pages/fr/getting-started.md]: Done

File: src/pages/fr/getting-started.md (hunk 4 above)
Comment (agriffard) [reply src/pages/fr/getting-started.md:104]: Ok, done.

**Review by MoustaphaDev — APPROVED** [review]
LGTM !

**Pull request merged**